org.envirocar.app/res/layout/activity_signin.xml

```diff
@@ -66,13 +66,14 @@
             android:id="@+id/activity_signin_username_input_layout"
             style="@style/Widget.MaterialComponents.TextInputLayout.OutlinedBox"
             android:layout_width="0dp"
-            android:layout_height="40dp"
+            android:layout_height="wrap_content"
             android:layout_marginTop="40dp"
             app:hintEnabled="false"
             app:layout_constraintEnd_toEndOf="@id/activity_signin_guideline_right"
             app:layout_constraintStart_toStartOf="@id/activity_signin_guideline_left"
             app:layout_constraintTop_toBottomOf="@+id/textView6"
-            app:layout_constraintWidth_max="500dp">
+            app:layout_constraintWidth_max="500dp"
+            app:endIconMode="custom">
```

Comment:
Added this to them which doesn't have any end icon so after making their height ``wrap_content``, they all still have a similar height. As the height changes to ``wrap_content`` after an error is shown, via this they all have same height even before error shows.